RevenueCatUI/Templates/LinTemplate5Step1View.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  File.swift
+//  
+//
+//  Created by Guillaume LAURES on 28/06/2024.
+//
+
+import Foundation
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct LinTemplate5Step1View: TemplateViewType {
+    static var paywallFirstStepImageBackgroundColor = Color(red: 255 / 255.0, green: 239 / 255.0, blue: 94 / 255.0)
+    static var bundle = Foundation.Bundle.module
+    let configuration: TemplateViewConfiguration
+    @Environment(\.userInterfaceIdiom)
+    var userInterfaceIdiom
+    @Environment(\.verticalSizeClass)
+    var verticalSizeClass
+    @Environment(\.horizontalSizeClass)
+    var horizontalSizeClass
+    @EnvironmentObject
+    private var introEligibilityViewModel: IntroEligibilityViewModel
+    @State
+    private var selectedPackage: TemplateViewConfiguration.Package
+    
+    var isEligibleToFreeTrial: Bool {
+        introEligibilityViewModel.allEligibility[selectedPackage.content] == .eligible
+    }
+    
+    let features = [
+        (title: localize("Step1.Feature1.Title", value: "No limits to your creativity"),
+         subtitle: localize("Step1.Feature1.Subtitle", value: "Export to vector formats and with any resolutions.")),
+        (title: localize("Step1.Feature2.Title", value: "Transform your photos instantly"),
+         subtitle: localize("Step1.Feature2.Subtitle", value: "Unlimited background removal usage and up to 50 AI Background generations")),
+        (title: localize("Step1.Feature3.Title", value: "Images to vector in seconds"),
+         subtitle: localize("Step1.Feature3.Subtitle", value: "Unlimited Auto-trace uses")),
+        (title: localize("Step1.Feature4.Title", value: "Unlimited files & artboards"),
+         subtitle: localize("Step1.Feature4.Subtitle", value: "Extend your design space"))
+    ]
+    
+    init(_ configuration: TemplateViewConfiguration) {
+        self._selectedPackage = .init(initialValue: configuration.packages.default)
+        self.configuration = configuration
+    }
+    
+    var body: some View {
+        SideBySideView {
+            titleFeaturesView
+        } rightView: {
+            auxiliaryDetailsView
+        }
+    }
+    
+    @ViewBuilder
+    private var titleFeaturesView: some View {
+        VStack {
+            scrollableView
+                .frame(maxWidth: .infinity)
+                .scrollableIfNecessaryWhenAvailable(enabled: configuration.mode.isFullScreen)
+            Spacer()
+            PurchaseButton(
+                packages: configuration.packages,
+                selectedPackage: selectedPackage,
+                configuration: configuration
+            ) {
+                print("Go to next step")
+            }
+            .frame(maxWidth: Constants.defaultContentWidth)
+            .padding([.bottom], 29)
+        }
+    }
+    
+    @ViewBuilder
+    private var scrollableView: some View {
```

Comment:
this is a bit confusing, as it's not actually scrollable. ContentView?

Reply:
I guess it could become scrollable on very small iPhone screens

Reply:
there is no scrolling here, you are adding it in the `titleFeaturesView`

Reply:
Oh ok I understand what you mean, fixed 👍